### PR TITLE
537 improve readme

### PR DESCRIPTION
**Pull request by sandeep-ps**
I'm creating this PR to update the current README. If there is a lot of review feedback, I can convert this to a draft and work on those changes.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -59,3 +59,15 @@ scripts/keycloak/data/*
...
+.env
+.venv
+env/
+venv/
+ENV/
+env.bak/
+venv.bak/
+
+# Tests
+backend/app/tests/
```
Comment (longshuicy) [inline .gitignore:73]: any reason we are not including tests to version control?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,13 +5,23 @@ All notable changes to this project will be documented in this file.
...
 - Use Keycloak for user management, JWT tokens and federated identity.
 - Initial implementation of file versioning and nested folders within datasets.
 - Initial implementation of metadata entries and metadata definitions.
 - Initial implementation of search using Elasticsearch.
 - Improvements to overall UI and UX. Standardized components and widgets used.
 - Helm charts for Kubernetes deployment.
+
+[unreleased]: https://github.com/clowder-framework/clowder2/compare/v2.0.0-alpha.2...HEAD
+
+[alpha-2]: https://github.com/clowder-framework/clowder2/releases/tag/v2.0.0-alpha.2
```
Comment (longshuicy) [inline CHANGELOG.md:27]: I like it how we include tags.. maybe we should start tagging more often? @lmarini

File: .gitignore (hunk 1 above)
Comment (sandeep-ps) [reply .gitignore:73]: Oh! I meant to ignore the additional outputs generated by tests. I will update this. Thanks.

Comment (sandeep-ps) [reply .gitignore]: Updated. Please see. Thanks.

**Review by longshuicy — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (longshuicy) [reply .gitignore]: Looks good. Approved :-)

File: CHANGELOG.md (hunk 2 above)
Comment (lmarini) [reply CHANGELOG.md:27]: yes, we should

**Pull request merged**